src/section.h

```diff
@@ -59,7 +59,7 @@ class SectionInfo
     SectionInfo(const QCString &label, const QCString &fileName, int lineNr,
                 const QCString &title, SectionType type, int level,const QCString &ref) :
         m_label(label), m_title(title), m_type(type), m_ref(ref),
-        m_lineNr(lineNr), m_fileName(fileName), m_level(level)
+        m_lineNr(lineNr), m_fileName(fileName), m_level(level), m_generated(false), m_definition(nullptr)
```

Comment:
I don't think this is an improvement. Data member initialization (added in C++11) is actually preferred over old style constructor initialization. See also https://www.cppstories.com/2015/02/non-static-data-members-initialization/ for a more elaborate discussion about the pros and cons.